packages/desktop/components/popup/popups/CheckForUpdatesPopup.svelte

```diff
@@ -17,73 +17,81 @@
     }
 
     function onVisitDownloadsClick(): void {
-        openUrlInBrowser('https://firefly.iota.org')
+        openUrlInBrowser('https://bloomwallet.io/')
     }
 
     function onCloseClick(): void {
         closePopup()
     }
 
+    let continueButton: ButtonProps
+    function setContinueButton(): void {
+        if (hasAutoUpdate && !$appVersionDetails.upToDate) {
+            continueButton = {
+                onClick: onDownloadClick,
+                disabled: $appUpdateState.busy,
+                text: localize('actions.updateBloom'),
+            }
+        } else if (!$appVersionDetails.upToDate) {
+            continueButton = {
+                onClick: onVisitDownloadsClick,
+                text: localize('actions.viewDownloads'),
+            }
+        } else {
+            return undefined
+        }
+    }
+
     onMount(() => {
         if (process.env.NODE_ENV !== 'development') {
             checkForAppUpdate()
         }
         hasAutoUpdate = features.electron.autoUpdate[OS]?.enabled
```

Comment:
we can do this outside of the `onMount`

In line 12:
```suggestion
        const hasAutoUpdate = features.electron.autoUpdate[OS]?.enabled
```